clang/test/CodeGenSYCL/nvptx-short-ptr.cpp

```diff
@@ -0,0 +1,10 @@
+// RUN: %clang_cc1 -fsycl-is-device -disable-llvm-passes \
```

Comment:
Can you add a comment describing what you're checking for in the layout here?

Reply:
done, thanks